### ISSUE TITLE
Improvement suggestions

### ISSUE THREAD

**Issue by rapperskull** [hagronnestad/com-port-proxy#2]
Hi @hagronnestad, I discovered this tool some days ago and it does exactly what I needed. I have just a few suggestions to improve it:

- Continously scan for COM ports. Now when the program starts, it scans for COM ports only once. If you add or remove ports, you have to restart the program. It would be nice to have a button to trigger a port rescan, if not continously scanning for new ports.
- Disconnect when closing the window. Right now, if you close the window, the program will still be running in the background, keeping the ports busy and consuming a lot of CPU.
- Make a release, so that people can simply download the program, without having to compile it themselves.

**Comment by hagronnestad** [comment]
Hi @rapperskull, these are all good suggestions. Feel free to implement any changes and send a pull request my way. I agree that a release would be really helpful, I'll be setting that up.
